webapp-django/crashstats/home/static/home/js/home.js

```diff
@@ -0,0 +1,386 @@
+/*global MG: true, socorro: true */
+
+$(function () {
+    'use strict';
+
+    // A polyfill for `endsWith`
+    if (!String.prototype.endsWith) {
```

Comment:
Let's avoid globals as much as we possibly can. If a particular page needs a particular polyfill, we put in in that app's static dir. If we find we need them repeatedly, we move it to the `base`'s static dir. 

Global solutions are very hard to undo as the project grows. Exceptions might be jquery or d3 .